qiskit/algorithms/minimum_eigen_solvers/vqe.py

```diff
@@ -337,7 +385,10 @@ def print_settings(self):
         else:
             ret += "ansatz has not been set"
         ret += "===============================================================\n"
-        ret += f"{self._optimizer.setting}"
+        if callable(self.optimizer):
+            ret += "Optimizer is custom callble"
+        else:
+            ret += f"{self._optimizer.setting}"
         ret += "===============================================================\n"
         return ret
```

Comment:
This section looks like it's missing newlines before the `"====="` bits.

Reply:
indeed it is!